README.md

```diff
@@ -1,84 +1,80 @@
-  * Download `celo-monorepo` branch `rc1` and `yarn && yarn build`
-  * Download `celo-blockchain` branch `rc1-tracing-fix` and `make all`
-  * Download `rosetta` branch `master` update go.mod and `make gen-contracts && make all`
-  * Run `make rc1-env` to create an empty datadir with the genesis block
+For an understanding of inputs & outputs check [servicer.go](./service/rpc/servicer.go)
 
-```bash
-rosetta run \
-  --genesis ./envs/rc1/genesis.json \
-  --geth ../celo-blockchain/build/bin/geth \
-  --staticNode "enode://5e0f4e3aaa096e2a2db76622b335cab4d3224d08d16cb11e8855a3a5f30c19d35d81a74b21271562e459495ab203c2f3a5a5747a83eb53ba046aeeb09aa240ff@34.83.110.24:30303"
-  --datadir "./envs/rc1"
-```
+## Running Rosetta Docker Image
 
-### Example for Alfajores:
+Rosetta is released as a docker image: `us.gcr.io/celo-testnet/rosetta`. All version can be found on the [registry page](https://us.gcr.io/celo-testnet/rosetta)
 
-Prerequisites:
-  * Download `celo-monorepo` branch `alfajores` and `yarn && yarn build`
-  * Download `celo-blockchain` branch `alfajores-tracing-fix` and `make all`
-  * Download `rosetta` branch `master` update go.mod and `make gen-contracts && make all`
-  * Run `make alfajores-env` to create an empty datadir with the genesis block
+Within the docker image, we pack `rosetta` binary and also `geth` binary from celo-blockchain. Rosetta will run both.
 
-```bash
-rosetta run \
-  --genesis ./envs/alfajores/genesis.json \
-  --geth ../celo-blockchain/build/bin/geth \
-  --staticNode "enode://05977f6b7d3e16a99d27b714f8a029a006e41ec7732167d373dd920d31f72b3a1776650798d8763560854369d36867e9564dad13b4b60a90c347feeb491d83a9@34.83.42.50:30303"
-  --datadir "./envs/alfajores"
-```
+### Configuration
 
-### Example for RC0:
+The necessary parameters to run rosetta are:
 
-Prerequisites:
-  * Download `celo-monorepo` branch `rc0` and `yarn && yarn build`
-  * Download `celo-blockchain` branch `mc/rosetta-rc0` and `make all`
-  * Download `rosetta` branch `rc0` update go.mod and `make gen-contracts && make all`
-  * Run `make rc0-env` to create an empty datadir with the genesis block
+ * `genesis.json` for the target network (can be found by `curl 'https://storage.googleapis.com/genesis_blocks/baklava' > genesis.json`)
+ * `staticNode` enodeURL. Rosetta will directly peer to the list of staticNode provided (it doens't run discovery protocol for the moment).
+    This node can be any you have access to. For a public list check `https://storage.cloud.google.com/static_nodes/baklava`
 
-```bash
-rosetta run \
-  --genesis ./envs/rc0/genesis.json \
-  --geth ../celo-blockchain/build/bin/geth \
-  --staticNode "enode://33ac194052ccd10ce54101c8340dbbe7831de02a3e7dcbca7fd35832ff8c53a72fd75e57ce8c8e73a0ace650dc2c2ec1e36f0440e904bc20a3cf5927f2323e85@34.83.199.225:30303" \
-  --datadir "./envs/rc0"
-```
+Additionaly, it needs a data directory for the geth datadir & rosetta.db
+
+### Run
 
-## Running with the docker image
+To run rosetta:
```

Comment:
sometimes Rosetta is caps-cased, other times not -- let's choose one and stick w it?